src/app/saga.test.ts

```diff
@@ -225,9 +245,9 @@ describe('handleDeepLink', () => {
   })
 
   it('Handles short share deep link', async () => {
```

Comment:
If we're sure we don't need to resolve them ourselves,
```suggestion
  it('Handles share deep link', async () => {
```